helm/kanister-operator/values.yaml

```diff
@@ -28,6 +33,14 @@ bpValidatingWebhook:
     mode: auto # If set to `custom` then secretName and caBundle should be provided
     secretName: '' # An already created Secret in kanister controller namespace having tls cert details
     caBundle: '' # A valid, CA bundle which is a PEM-encoded CA bundle for validating the webhook's server certificate
+repositoryServerController:
+  enabled: false
+  # startTimeout is used to specify the time in seconds to wait for starting the kopia repository server
+  # Default timeout is 600 seconds
+  serverStartTimeout:
+  logLevel: ''
+  container:
+    name: ''
```

Comment:
Let's set the default values instead of keeping them blank. So that the default values will be rendered to generated helm chart